src/client/datascience/kernel-launcher/kernelEnvVarsService.ts

```diff
@@ -65,6 +65,7 @@ export class KernelEnvironmentVariablesService {
             return kernelEnv;
         }
         // Merge the env variables with that of the kernel env.
+        const hasInterpreterEnv = interpreterEnv != undefined;
```

Comment:
Looks like you need to refresh your branch. This shouldn't be different. I already submitted it to main

Reply:
I do not understand why this is showing up. The PR is into origin/main, but somehow it doesn't have an up-to-date version of that branch?

Reply:
Was your local main up to date? I think it's a PR for merging release into an older main branch that you had, not the current main head.

Reply:
Pretty sure the issue was that Github is using the common ancestor for comparison between the two branches: https://stackoverflow.com/a/35203417/1253265
I went ahead and merged and confirmed that there are no new commits to this file in main.